File: conformance-tests/statements/declarations/variable/function-return-type-with-comments_m5.ts
const foo: ()=> /* Comment before foo void */ void /* Comment after foo void */= ():void => {}
const fizz = function ():/* Comment before fizz void */void/* Comment after fizz void */ {}
const buzz: ()=>/* Comment before lhs buzz void */void/* Comment after lhs buzz void */ = function():/* Comment before rhs buzz void */void/* Comment after rhs buzz void */ {}
function jazz():/* Comment before jazz void */void/* Comment after jazz void */ {}

File: conformance-tests/statements/declarations/variable/function-return-type-with-comments_m5.lua
-- ROBLOX upstream: https://github.com/Roblox/js-to-lua/blob/sha/conformance-tests/statements/declarations/variable/function-return-type-with-comments_m5.ts
local foo: () ->  --[[ Comment before foo void ]]() --[[ Comment after foo void ]]
function foo(): () end
local function fizz():  --[[ Comment before fizz void ]]()
	--[[ Comment after fizz void ]]
end
local buzz: () ->  --[[ Comment before lhs buzz void ]]() --[[ Comment after lhs buzz void ]]
function buzz():  --[[ Comment before rhs buzz void ]]()
	--[[ Comment after rhs buzz void ]]
end
local function jazz():  --[[ Comment before jazz void ]]()
	--[[ Comment after jazz void ]]
end
